feat(blog) Upgrades blog & articles (#1468)
solana-foundation/solana-com: apps/media/content/upgrades/reduced-slot-times.mdx

---
title: Reduced Slot Times
description: >-
  Solana is reducing slot times from 400ms to 200ms (SIMD-0525), cutting
  latency in half for users and tightening spreads for market makers, rolled
  out in phased 50ms decrements.
subtitle: Cutting slot times from 400ms to 200ms
publishedAt: 2026-06-09T00:00:00.000Z
status: published
author: solana-foundation
badges:

>   - text: 🟡 Under Development

Reviewer: ```suggestion
  - text: Q3 2026
```

    color: yellow
    variant: badge
metrics:
  - value: 200ms
    label: New slot time, down from 400ms
  - value: 2x
    label: Faster confirmations for users
categories:
  - category: upgrades
tags:
  - tag: announcements
---

# Reduced Slot Times

> **TBD** • Solana Foundation

Reviewer: ```suggestion
Q3 2026 • Solana Foundation
```

Solana will reduce its slot times from 400ms to 200ms. This change was first
discussed in
[SIMD Discussion 469](https://github.com/solana-foundation/solana-improvement-documents/discussions/469)
and formally proposed and approved in
[SIMD-0525](https://github.com/solana-foundation/solana-improvement-documents/pull/525).
This change will make the protocol more competitive in terms of latency. It takes
advantage of the performance improvements made to Solana validator clients
especially in Turbine, and Replay.

This also doubles as a censorship resistance measure since it narrows the time a
leader has a monopoly over block building. Together with the separately proposed
[reduction of consecutive leader slots](https://github.com/solana-foundation/solana-improvement-documents/pull/498),

> this is a marked improvement in fairness for both validators and users alike.

Reviewer: ```suggestion
this improves fairness for validators and users by limiting individual leader control windows.
```

|                                  |     |
| -------------------------------- | --- |

> | Expected Mainnet Activation Date | TBD |

Reviewer: ```suggestion
| Expected Mainnet Activation Date | Q3 2026 |
```

> | Devnet Activation                | TBD |

Reviewer: ```suggestion
| Devnet Activation                | Q3 2026 |
```
